README.md

```diff
@@ -77,55 +82,90 @@ version, and in terms of non-breaking changes it includes:
+    * A `fiat_u64` backend using formally verified field arithmetic from [fiat-crypto].
+
+Architecture selection within serial and formally verified backends is
+automatic. E.g., if you're compiling with the serial backend on a 64-bit
+machine, then the `u64` backend is automatically chosen. And if the
+`fiat_backend` feature is set, then the `fiat_u64` backend is chosen.
+
+If you need a 32-bit backend on an x86-64 Linux machine, then simple
+cross-compiling will work:
 
-The `std` feature is enabled by default, but it can be disabled for no-`std`
-builds using `--no-default-features`.  Note that this requires explicitly
-selecting an arithmetic backend using one of the `_backend` features.
-If no backend is selected, compilation will fail.
+* `sudo apt install gcc-multilib` (or whatever package manager you use)
+* `rustup target add i686-unknown-linux-gnu`
+* `cargo build --target i686-unknown-linux-gnu`
 
-## Backend selection
+# Documentation
 
-Backend selection is done automatically. E.g., if you're compiling on a
-64-bit machine, then the `u64` backend is automatically chosen. And
-if the `fiat_backend` feature is set, then the fiat `u64` backend is
-chosen.
+The semver-stable, public-facing `curve25519-dalek` API is documented [here][docs].
 
-If you need a `u32` backend on a `u64` machine, then simple
-cross-compiling will work on an x86-64 Linux machine:
+## Building Docs Locally
 
-* `sudo apt install gcc-multilib` (or whatever package manager you use)
-* `rustup target add i686-unknown-linux-gnu`
-* `cargo build --target i686-unknown-linux-gnu`
+The `curve25519-dalek` documentation requires a custom HTML header to include
+KaTeX for math support. Unfortunately `cargo doc` does not currently support
+this, but docs can be built using
+```sh
+make doc
+```
+for regular docs, and
+```sh
+make doc-internal
+```
+for docs that include private items.
+
+# Maintenance Policies
+
+* All on-by-default features of this library are covered by [semantic versioning][semver] (SemVer).
+* SemVer exemptions are outlined below for MSRV and public API.
+
+## Minimum Supported Rust Version
+
+| Releases | MSRV   |
+| :---     | :---   |
+| 4.x      | 1.56.1 |
+| 3.x      | 1.41.0 |
+
+From 4.x and on, MSRV changes will be accompanied by a minor version bump.
 
-# Minimum Supported Rust Version
+## Public API SemVer Exemptions
 
-This crate requires Rust 1.56.1 at a minimum. 3.x releases of this crate supported an MSRV of 1.41.
+Breaking changes to SemVer exempted components affecting the public API will be accompanied by
+_some_ version bump. Below are the specific policies:
 
-In the future, MSRV changes will be accompanied by a minor version bump.
+| Releases | Public API Component(s)               | Policy              |
+| :---     | :---                                  | :---                |
+| 4.x      | Dependencies `digest` and `rand_core` | Minor SemVer bump   |
 
```

Comment:
I removed the line here about removing exemptions in the future. It wasn't saying much and we don't need to make those promises now imo.